content/docs/1.13/performance-tuning.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Performance Tuning Guide
+description: Tweaking your Jaeger instance to achieve a better performance
+weight: 10
+---
+
+Jaeger was built from day 1 to be able to ingest huge amounts of data in a resilient way. To better utilize resources that might cause delays, such as storage or network communications, Jaeger buffers and batches data. When more spans are generated than Jaeger is able to safely process, spans might get dropped. However, the defaults might not fit all scenarios: for instance, agents running as a sidecar might have more memory constraints than agents running as a daemon in bare metal.
+
+## Adjusting the client settings
+
+### Settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`
```

Comment:
using env names in the heading makes the sidebar menu look rather ugly (one even gets cut off). 

![image](https://user-images.githubusercontent.com/3523016/61314929-b34edb00-a7cb-11e9-885b-c25795fbfcbb.png)


I would recommend

### Sampling configuration

Sampling in the client can be controlled with `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM` environment variables. [...]